pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/FilterOperator.java

```diff
@@ -46,14 +48,14 @@
 public class FilterOperator extends BaseOperator<TransferableBlock> {
   private static final String EXPLAIN_NAME = "FILTER";
   private final Operator<TransferableBlock> _upstreamOperator;
-  private final FilterOperand _filterOperand;
+  private final TransformOperand _filterOperand;
```

Comment:
Not sure I follow this change. 

This makes it look like that operand will be a function based evaluation which may not be necessarily true ?